src/mat/4C_mat_fourier.hpp

```diff
@@ -19,74 +16,58 @@
 
 FOUR_C_NAMESPACE_OPEN
 
-/*----------------------------------------------------------------------*
- |                                                           dano 09/09 |
- *----------------------------------------------------------------------*/
 namespace Mat
 {
   namespace PAR
   {
-    /*----------------------------------------------------------------------*/
-    /// material parameters for FourierIso material
+    /// material parameters for Fourier material
     ///
     /// <h3>Input line</h3>
-    /// MAT 1 THERM_FourierIsoIso CAPA 1.0 COND 1.0
-    class FourierIso : public Core::Mat::PAR::Parameter
+    /// MAT 1 MAT_Fourier CAPA 1.0 NUMSPACEDIM 3 CONDUCT 1.0 0.1 10.0
+    class Fourier : public Core::Mat::PAR::Parameter
     {
      public:
       /// standard constructor
-      FourierIso(const Core::Mat::PAR::Parameter::Data& matdata);
+      Fourier(const Core::Mat::PAR::Parameter::Data& matdata);
 
       /// @name material parameters
       //@{
 
       /// volumetric heat capacity
       const double capa_;
+
       /// heat conductivity
-      const double conduct_;
+      const std::vector<double> conduct_;
+
+      /// number of conductivity components
+      const int conduct_para_num_;
 
       //@}
 
       /// create material instance of matching type with my parameters
       std::shared_ptr<Core::Mat::Material> create_material() override;
-
-    };  // class FourierIso
-
+    };
   }  // namespace PAR
 
-  class FourierIsoType : public Core::Communication::ParObjectType
+  class FourierType : public Core::Communication::ParObjectType
   {
    public:
-    std::string name() const override { return "FourierIsoType"; }
+    std::string name() const override { return "FourierAnisoType"; }
 
-    static FourierIsoType& instance() { return instance_; };
+    static FourierType& instance() { return instance_; };
 
     Core::Communication::ParObject* create(Core::Communication::UnpackBuffer& buffer) override;
 
    private:
-    static FourierIsoType instance_;
+    static FourierType instance_;
   };
 
-  /*----------------------------------------------------------------------*/
-  /// FourierIso material according to [1]
-  ///
-  /// This is a FourierIso's law of isotropic, instationary heat conduction
-  ///
-  /// <h3>References</h3>
-  /// <ul>
-  /// <li> [1] GA Holzapfel, "Nonlinear solid mechanics", Wiley, 2000.
-  /// </ul>
-  ///
-  /// \author dano
-  /// \date 09/09
```

Comment:
I would suggest to keep this reference, and add `The anisotropic version is the extension of isotropic case with thermal conductivity treated as tensor.`

Reply:
Added the documentation again.